data.md

```diff
@@ -53,10 +53,10 @@ Some important numbers that are referred to often during execution:
 -   `chop` interperets an integers modulo $2^256$.
 
 ```{.k .uiuck .rvk}
-    syntax Int ::= chop ( Int ) [function]
- // --------------------------------------
-    rule chop ( I:Int ) => I %Int pow256 requires I <Int 0  orBool I >=Int pow256
-    rule chop ( I:Int ) => I             requires I >=Int 0 andBool I <Int pow256
+    syntax Int ::= chop ( Int ) [function, smtlib(chop)]
+ // ----------------------------------------------------
+    rule chop ( I:Int ) => I %Int pow256 requires I  <Int 0  orBool I >=Int pow256    [concrete]
```

Comment:
what does `concrete` mean?  a comment there?

Reply:
we have a discussion on this, but we think the `concrete` attribute is k-specific, so better to have a document somewhere else rather than under evm semantics.

Reply:
btw, `concrete` means that the functional rule is only applied when all of the arguments are concrete not symbolic.

this helps to prevent the functional term to keep expanding without being revolved, which in turn helps both debugging, performance, and writing the verification lemmas in a succinct and readable way.